src/create-emotion/index.js

```diff
@@ -63,7 +63,7 @@ type EmotionOptions = {
 }
 
 function createEmotion(
-  context: {__SECRET_FANCY_EMOTION__?: Emotion},
+  context: { __SECRET_FANCY_EMOTION__?: Emotion },
```

Comment:
What would a "Secret Fancy Emotion" be? "Elated", "Discombobulated"? Don't mind me it's Friday

Reply:
Hahaha... it's a secret ;)

For context... By default, Emotion generates a global variable called `__SECRET_EMOTION__`. We had an issue where a Fancy powered app (Beacon2) rendered into a user environment that happened to use Emotion as well.

Because of this... our modified version of Emotion broke their Emotion 😅 

So! To prevent this from happening, we created our own `__SECRET_FANCY_EMOTION__` namespace :)